### PR TITLE
Enable 2.13 build steps

### PR DESCRIPTION
**Pull request by mijicd**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -48,13 +48,14 @@ lazy val keeper = project
   .settings(stdSettings("zio-keeper"))
   .settings(
     libraryDependencies ++= Seq(
-      "dev.zio"     %% "zio"             % "1.0.0-RC17",
```
Comment (mschuwalow) [inline build.sbt]: We should start introducing a constant for zio versions. :D

File: membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala [hunk 2]
```diff
@@ -33,8 +30,7 @@ object TransportSpec
               bind(addr).take(1).runCollect.map(_.head)
 
             environment >>> Live.live(for {
-              port   <- freePort
-              addr   <- SocketAddress.inetSocketAddress(port)
+              addr   <- SocketAddress.inetSocketAddress(0)
```
Comment (mschuwalow) [inline membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala]: Nice fix!

File: build.sbt (hunk 1 above)
Comment (pshemass) [reply build.sbt]: I added this to https://github.com/zio/zio-keeper/issues/115

File: membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala (hunk 2 above)
Comment (mijicd) [reply membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala]: Still fiddle around it, should be done tonight.

**Pull request merged**